### DISCUSSION TITLE
Arcane Explosion for Fire and Frost?

### DISCUSSION THREAD

**Discussion by SARCaelvin** [Hekili/hekili#993] [Q&A]
I am really confused how these two specs have the spell in the rotation part of the addon. I could not find it as a part of the rotation on any guide, should this be ticketed as bug?

**Comment by Hekili** [comment]
Legendary.

**Reply by SARCaelvin** [reply]
Even when you don't have the one where you have to use all the schools?

**Reply by Hekili** [reply]
Yes, there are scenarios without the legendary as well.  Fire Mage without Flame Patch, for example.  You can review the priorities directly to try to break it down.

```
# APL Variable Option: This variable specifies the number of targets at which Arcane Explosion outside of Combustion should be used.
actions.precombat+=/variable,name=arcane_explosion,op=set,if=variable.arcane_explosion=0,value=99*talent.flame_patch+2*!talent.flame_patch
# APL Variable Option: This variable specifies the percentage of mana below which Arcane Explosion will not be used.
actions.precombat+=/variable,name=arcane_explosion_mana,default=40,op=reset
```

```
actions.standard_rotation+=/arcane_explosion,if=active_enemies>=variable.arcane_explosion&mana.pct>=variable.arcane_explosion_mana
```

This breaks down to Arcane Explosion used with 99 targets if you have Flame Patch (i.e., you don't) or 2 targets without Flame Patch, as long as you have 40% mana or more.  (Remember, that this is a priority list, so we only get here if you don't have something better to cast that's higher in the priority list.)
